modules/odc-connecting-components.adoc

```diff
@@ -83,7 +88,7 @@ spec:
 ----
 .. Click *Create* to create the `OperatorSource` in your cluster.
 . Install the *PostgreSQL Database* Operator:
-.. In the *Administrator* perspective of the console, navigate to the *Operators -> OperatorHub*.
+.. In the *Administrator* perspective of the console, navigate to *Operators -> OperatorHub*.
```

Comment:
@Preeticp the first point of the procedure can be more specific since the user should not get confused and think only by using the particular operator source we can install db operators. I think we can change it to
`Add PostgresSQL operator provided by Red Hat to OperatorHub using the following OperatorSource. A backing OperatorSource  exposes the binding information in secrets, ConfigMaps, status, and spec attributes`